.eslintrc.js

```diff
@@ -65,7 +65,19 @@ module.exports = {
     'no-restricted-imports': [
       'error',
       {
-        patterns: ['@mui/*/*/*'],
+        patterns: [
+          {
+            group: [
+              '@mui/*/*/*',
+              '@pigment-css/*/*/*',
+              '@base_ui/*/*/*',
+              // Allow any import depth with any internal packages
+              '!@mui/internal-*/**',
+              '!@mui/docs/**',
```

Comment:
We should rename this one to `@mui/internal-docs`. `@mui/docs` was exporting public packages, it's not the case anymore.